### PR TITLE
IBX-6117: Ibexa Connect scenario block added in Developer Documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-4961)
| Versions      | master
| Edition       | Connect, Experience

Ibexa Connect scenario block added in Developer Documentation.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Code samples are working
- [x] PHP code samples have been fixed with PHP CS fixer
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,5 +20,10 @@
     "scripts": {
         "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.php -v --show-progress=dots",
         "check-cs": "@fix-cs --dry-run"
+    },
+    "config": {
+        "allow-plugins": {
+            "php-http/discovery": true
+        }
```
Comment (Steveb-p) [inline composer.json:27]: We try to explicitly limit plugins executed during composer installation, so we disable them by default.
```suggestion
        "allow-plugins": false
```

File: docs/content_management/pages/ibexa_connect_scenario_block.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+description: Work with Ibexa Connect scenario block that retrieves and displays data from an Ibexa Connect webhook. 
+---
+
+# Ibexa Connect scenario block
+
+Ibexa Connect scenario block retrieves and displays data from an Ibexa Connect webhook. 
+
+Scenario block is regular [Page block](page_blocks.md) and can be configured on field definition level as any other block.
```
Comment (Steveb-p) [inline docs/content_management/pages/ibexa_connect_scenario_block.md:9]:
```suggestion
Scenario block is a regular [Page block](page_blocks.md) and can be configured on field definition level as any other block.
```

**Review by Steveb-p — APPROVED** [review]

**Comment by MagdalenaZuba** [comment]
This is a procedure with steps, consider placing all code in one directory and only linking to needed samples. Thanks to that, it will be easier to test in the future.

Do we need QA on that?

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: docs/content_management/img/ibexa_connect_design_tab.png
Comment (juskora) [inline docs/content_management/img/ibexa_connect_design_tab.png:1]: I think the screenshot should show actual values in the View drop-down: company_customers and External clients.

File: docs/content_management/pages/ibexa_connect_scenario_block.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+At this point the Ibexa Connect scenario block is ready to be used in Page Builder.
+
+### Configure Ibexa Connect scenario block in Page Builder
+
+Now, you can configure Ibexa Connect scenario block in Page Builder.
+
+To do it, in your Page add Ibexa Connect block by dragging it from the menu to a drop zone and enter block settings. 
+
+- In the **Basic** tab in **Webhook link** field, provide a link to an Ibexa Connect webhook, 
+for example, `https://ibexa.integromat.celonis.com/3/scenarios/688/edit`:
```
Comment (juskora) [inline docs/content_management/pages/ibexa_connect_scenario_block.md:96]: As Connect has changed the URL address, should we change it here to `https://connect.ibexa.co/3/scenarios/688/edit`? 
@bdunogier what do you think?

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/content_management/pages/ibexa_connect_scenario_block.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+![Ibexa Connect Basic tab](ibexa_connect_basic_tab.png)
+
+- In the **Design** tab, choose one of declared templates, in the following example, `company_customers` or `External clients`. 
+To do it, extend drop-down list in the **View** field and choose one of the available options.
+
+![Ibexa Connect Design tab](ibexa_connect_design_tab.png)
+
+Click **Submit** button to confirm.
+
+After submitting the block, Page refreshes and Ibexa Connect block displays data from provided Ibexa Connect webhook.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/ibexa_connect_scenario_block.md:72]:
```suggestion
Click **Submit** button to confirm.
After submitting the block, Page refreshes and Ibexa Connect block displays data from provided Ibexa Connect webhook. 
```

File: docs/content_management/pages/ibexa_connect_scenario_block.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+---
+description: Work with Ibexa Connect scenario block that retrieves and displays data from an Ibexa Connect webhook. 
+---
+
+# Ibexa Connect scenario block
+
+Ibexa Connect scenario block retrieves and displays data from an Ibexa Connect webhook. 
+
+Scenario block is a regular [Page block](page_blocks.md) and can be configured on field definition level as any other block.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/ibexa_connect_scenario_block.md:9]:
```suggestion
Ibexa Connect scenario block retrieves and displays data from an Ibexa Connect webhook. 
Scenario block is a regular [Page block](page_blocks.md) and can be configured on the field definition level as any other block.
```

File: docs/content_management/pages/ibexa_connect_scenario_block.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+In the following example, the configuration of the block is non-complex - block is only used to display the content transferred from an Ibexa Connect webhook.
+
+At this point the Ibexa Connect scenario block is ready to be used in Page Builder.
+
+### Configure Ibexa Connect scenario block in Page Builder
+
+Now, you can configure Ibexa Connect scenario block in Page Builder.
+
+To do it, in your Page add Ibexa Connect block by dragging it from the menu to a drop zone and enter block settings.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/ibexa_connect_scenario_block.md:58]:
```suggestion
Now, you can configure Ibexa Connect scenario block in Page Builder.
To do it, in your Page add Ibexa Connect block by dragging it from the menu to a drop zone and enter block settings. 
```

File: docs/content_management/pages/ibexa_connect_scenario_block.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+Then, in `templates/block` directory under `default.html.twig`, provide your block configuration:
+
+```html+twig
+[[= include_file('code_samples/page/ibexa_connect_scenario_block/templates/block/default.html.twig') =]]
+```
+
+In the following example, the configuration of the block is non-complex - block is only used to display the content transferred from an Ibexa Connect webhook.
+
+At this point the Ibexa Connect scenario block is ready to be used in Page Builder.
```
Comment (MagdalenaZuba) [inline docs/content_management/pages/ibexa_connect_scenario_block.md:52]:
```suggestion
In the following example, the configuration of the block is non-complex - block is only used to display the content transferred from an Ibexa Connect webhook.
At this point the Ibexa Connect scenario block is ready to be used in Page Builder.
```

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/content_management/pages/ibexa_connect_scenario_block.md (hunk 3 above)
Comment (bdunogier) [reply docs/content_management/pages/ibexa_connect_scenario_block.md]: Yes, please ! All celonis.com URLs must be updated to connect.ibexa.co.

**Review by juskora — APPROVED** [review]

**Pull request merged**